src/internal/umd.ts

```diff
@@ -0,0 +1,22 @@
+/*
+  NOTE: This is the global export file for rxjs v6 and higher.
+ */
+
+/* rxjs */
+export * from '../';
+
+/* rxjs/operators */
+import * as _operators from '../operators';
+export const operators = _operators;
```

Comment:
why not just `export * from '../operators';`? (also all the once below)

Reply:
otherwise this change LGTM. I think consistency is important, and making umd/globals usage as close to ESM usage will make documentation and learning of RxJS so much easier.